proto/storage/image_integration.proto

```diff
@@ -131,11 +135,13 @@ message ScannerV4Config {
   string matcher_endpoint = 3;
 }
 
+// Azure container registry configuration. Used by integrations of type "azure".
 message AzureConfig {
-  string endpoint = 1; // @gotags: scrub:"dependent" validate:"nolocalendpoint"
```

Comment:
Why `dependent` was removed?

Reply:
See my comment below https://github.com/stackrox/stackrox/pull/13635#discussion_r1937050945. I put `dependent` tag back and added a note in the UI that when WIF is enabled, any existing credentials are overwritten.